cmd/bosun/web/static/js/bosun.js

```diff
@@ -1636,6 +1636,9 @@ bosunControllers.controller('ExprCtrl', ['$scope', '$http', '$location', '$route
             }
         };
     }]);
+/// <reference path="angular.d.ts"/>
```

Comment:
did you upgrade typescript? I'm ok with it, but are there docs that need updating?